docs/apm-infra-correlation.md

```diff
@@ -0,0 +1,295 @@
+# How to Enable APM-Infra Correlation 
+
+In the APM Service Dashboards, there are charts that indicate the health of
+the underlying infrastructure. The default configuration of Splunk OpenTelemetry 
+Connector automatically configures this for you, but when specifying a custom 
+configuration it is important to keep the following in mind.
+
```

Comment:
Suggested change:

"The APM service dashboards include charts that indicate the health of the underlying infrastructure. The default configuration of Splunk OpenTelemetry Connector automatically configures this for you, but when specifying a custom configuration, it is important to keep the following in mind."

Also, when you say "configures this for you," are you referring to the dashboards?

Reply:
ty @tmorrisonsfx ! "configures this for you" - this was a suggestion by another reviewer, the connector's default configuration already contains the changes that we mention here.